site/en/blog/extension-news-october-2023/index.md

```diff
@@ -0,0 +1,151 @@
+---
+layout: 'layouts/blog-post.njk'
+title: What's happening in Chrome Extensions?
+description: >
+ An overview of the recent changes in Chrome Extensions, plus exciting upcoming extension features developers can look forward to.
+date: 2023-10-13
+authors:
+  - amysteam
+tags:
+  - extensions-news
+hero: 'image/BhuKGJaIeLNPW9ehns59NfwqKxF2/zHJnWcckEKIRCdXOGYNo.png'
+alt: >
+  What's happening in Chrome extensions?
+---
+
+Back in July, we launched [a new blog series](/blog/extension-news-july-2023/) to keep you up-to-date on extension developments. Thanks to your valuable feedback and our ongoing collaboration with fellow browser vendors in the WebExtensions Community Group, we continue to enhance extension APIs and work towards greater consistency across browsers.
+
+Welcome to the October edition! In this post, we'll look at some of the changes the Chrome extension team has made in the past few months, as well as some new features that'll come out later this year. Let's get started!
+
+{% Aside %}
+
+If you’d like a TL;DR, feel free to jump to the [Coming soon...](#coming-soon-apis) section for new extension features you can start looking forward to 🙂.
+
+{% endAside %}
+
+## New extension APIs and features {: #new-apis }
+
+In this section, we share some significant API launches, briefly review other API improvements, and share upcoming releases. All launches are currently available in the latest Beta release. See the [chromium release schedule](https://chromiumdash.appspot.com/schedule) for details.
+
+
+### Highlights {: #apis-highlights }
+
+#### Resolved known issues {: #closing-gap }
+
+The extension team has been actively working to resolve Manifest V3 stability issues. [Chrome 116 launched many improvements](/blog/chrome-116-beta-whats-new-for-extensions/) that helped us make significant progress toward closing the feature gap between Manifest V2 and V3. In Chrome 120, we will have finished addressing all our prioritized platform gaps and closed all critical bugs that are documented on the [known issues page](/docs/extensions/migrating/known-issues/). All features are currently available in Chrome 120 Canary, except fileHandler support for ChromeOS Lacros, which will land later this month. For more details, check out the updated [known issues page](/docs/extensions/migrating/known-issues/).
```

Comment:
```suggestion
The extension team has been actively working to resolve Manifest V3 stability issues. [Chrome 116 launched many improvements](/blog/chrome-116-beta-whats-new-for-extensions/) that helped us make significant progress toward closing the feature gap between Manifest V2 and V3. In Chrome 120, we will have finished addressing all our prioritized platform gaps and closed all critical bugs that are documented on the [known issues page](/docs/extensions/migrating/known-issues/). All features are currently available in Chrome 120 Canary, except fileHandler support for ChromeOS Lacros, and the userScripts API which will land later this month. Check out the updated [known issues page](/docs/extensions/migrating/known-issues/) for more details.
```

Reply:
@sebastianbenz WDTY? If you want to change it further LMK

Reply:
one nit: s/In Chrome 120, we will have finished addressing/In Chrome 120, we plan to finish addressing/